sendinblue.php

```diff
@@ -2993,6 +2996,222 @@ public function mailSendBySmtp()
+			</tr>
+			</thead>';
+
+		$this->_html_block_position .= '
+		<tr><td colspan="4"><form method="post" action="'.Tools::safeOutput($_SERVER['REQUEST_URI']).'">
+		<div>
+			<label>
+			'.$this->l('Right column block').'
+			</label>
+			<span class="'.$this->cl_version.'">
+			<input type="radio" class="rightColumn radio_nospaceing" id="yesradio" name="rightColumn" value="1"
+			'.(Configuration::get('Sendin_RightColumn', '', $this->id_shop_group, $this->id_shop) == 1 ? 'checked="checked" ' : '').'/>'.$this->l('Show').'
+			<input type="radio" class="rightColumn radio_spaceing2" id="noradio"
+			name="rightColumn" value="2" '.(Configuration::get('Sendin_RightColumn', '', $this->id_shop_group, $this->id_shop) == 2 ? 'checked="checked" ' : '').'/>'
+			.$this->l('Hide').'
+		</div>	
+		<div>
+			<label>
+			'.$this->l('Left column block').'
+			</label>
+			<span class="'.$this->cl_version.'">
+			<input type="radio" class="leftColumn radio_nospaceing" id="yesradio" name="leftColumn" value="1"
+			'.(Configuration::get('Sendin_LeftColumn', '', $this->id_shop_group, $this->id_shop) == 1 ? 'checked="checked" ' : '').'/>'.$this->l('Show').'
+			<input type="radio" class="leftColumn radio_spaceing2" id="noradio"
+			name="leftColumn" value="2" '.(Configuration::get('Sendin_LeftColumn', '', $this->id_shop_group, $this->id_shop) == 2 ? 'checked="checked" ' : '').'/>'
+			.$this->l('Hide').'
+		</div>
+		<div>
+			<label>
+			'.$this->l('Top block').'
+			</label>
+			<span class="'.$this->cl_version.'">
+			<input type="radio" class="top radio_nospaceing" id="yesradio" name="top" value="1"
+			'.(Configuration::get('Sendin_Top', '', $this->id_shop_group, $this->id_shop) == 1 ? 'checked="checked" ' : '').'/>'.$this->l('Show').'
+			<input type="radio" class="top radio_spaceing2" id="noradio"
+			name="top" value="2" '.(Configuration::get('Sendin_Top', '', $this->id_shop_group, $this->id_shop) == 2 ? 'checked="checked" ' : '').'/>'
+			.$this->l('Hide').'
+		</div>	
+		<div>
+			<label>
+			'.$this->l('Footer block').'
+			</label>
+			<span class="'.$this->cl_version.'">
+			<input type="radio" class="footer radio_nospaceing" id="yesradio" name="footer" value="1"
+			'.(Configuration::get('Sendin_Footer', '', $this->id_shop_group, $this->id_shop) == 1 ? 'checked="checked" ' : '').'/>'.$this->l('Show').'
+			<input type="radio" class="footer radio_spaceing2" id="noradio"
+			name="footer" value="2" '.(Configuration::get('Sendin_Footer', '', $this->id_shop_group, $this->id_shop) == 2 ? 'checked="checked" ' : '').'/>'
+			.$this->l('Hide').'
+		</div>
+		</div>
+		</td></tr></form>';
+			$option = '<option value="">'.$this->l('Select hook').'</option>';
+			$modules = Module::getModulesInstalled();
+			foreach ($modules as $module)
+			{
+				if ($module['name'] == 'sendinblue')
+				{
+					$module_data = array();
+					$module_data = $module;

```

Comment:
Because you do not need to iterate anymore of the array, you can add a ```break;``` when you found your module